src/UKHO.ERPFacade.API/Helpers/LicenceUpdatedSapMessageBuilder.cs

```diff
@@ -13,9 +13,8 @@ public class LicenceUpdatedSapMessageBuilder : ILicenceUpdatedSapMessageBuilder
         private readonly IFileSystemHelper _fileSystemHelper;
 
         public LicenceUpdatedSapMessageBuilder(ILogger<LicenceUpdatedSapMessageBuilder> logger,
```

Comment:
Could we move all of these message builder classes to a namespace called 'Messages' or something similar. Using the word 'helper' in class or namespace names should be avoided. The responsibility should be clear.

Reply:
Updated as suggested. Thanks